installing/validating-an-installation.adoc

```diff
@@ -0,0 +1,62 @@
+
+* See xref:../support/troubleshooting/troubleshooting-installations.adoc#querying-operator-status-after-installation_troubleshooting-installations[Querying Operator status after installation] for more information about querying Operator status if your installation is still progressing.
+
+* See xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-operator-issues[Troubleshooting Operator issues] for information about investigating issues with Operators.
+
+* See xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating a cluster between minor versions] for more information on updating your cluster.
+
+* See xref:../updating/updating-cluster-between-minor.adoc#understanding-upgrade-channels_updating-cluster-between-minor[OpenShift Container Platform upgrade channels and releases] for an overview about upgrade release channels.
+
+//Querying the status of the cluster nodes by using the CLI
+include::modules/querying-the-status-of-cluster-nodes-using-the-cli.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../support/troubleshooting/verifying-node-health.adoc#verifying-node-health[Verifying node health] for more details about reviewing node health and investigating node issues.
+
+//Reviewing the cluster status from the OpenShift Container Platform web console
+include::modules/reviewing-cluster-status-from-the-openshift-web-console.adoc[leveloffset=+1]
+
+//Reviewing the cluster status from the OpenShift Cluster Manager
+include::modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#using-insights-to-identify-issues-with-your-cluster[Using Insights to identify issues with your cluster] for more information about reviewing potential issues with your cluster.
+
+//Checking cluster resource availability and utilization
+include::modules/checking-cluster-resource-availability-and-utilization.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Understanding the monitoring stack] for more information about the {product-title} monitoring stack.
+
+//Listing alerts that are firing
+include::modules/listing-alerts-that-are-firing.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/managing-alerts.adoc#managing-alerts[Managing alerts] for further details about alerting in {product-title}.
+
+[id="validating-an-installation-next-steps"]
+== Next steps
+
+* See xref:../support/troubleshooting/troubleshooting-installations.adoc#troubleshooting-installations[Troubleshooting installations] if you experience issues when installing your cluster.
+
+* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-configuration-tasks[Post-installation machine configuration tasks]
```

Comment:
I think this should be written as a step, something like:
"To configure operating system features, see _Post-installation machine configuration tasks_."

Reply:
Thank you Bob for your review and for highlighting this. On reflection I have updated this next step to direct the reader to the 'Post-installation configuration -> Cluster tasks' assembly as follows:

* After installing {product-title}, you can further xref:../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[customize your cluster] to your requirements.